rust/tests/behaviour/mod.rs

```diff
@@ -54,8 +54,9 @@ pub struct Context {
     pub things: HashMap<String, Option<Thing>>,
     pub answer: Vec<ConceptMap>,
     pub answer_group: Vec<ConceptMapGroup>,
-    pub numeric_answer: Option<Numeric>,
-    pub numeric_answer_group: Vec<NumericGroup>,
+    pub fetch_answer: Option<JSON>,
+    pub value_answer: Option<Option<Value>>,
+    pub value_answer_group: Vec<ValueGroup>,
```

Comment:
A better way to store the latest query result would be to use an enum, but this is out of scope of this PR.

Reply:
#507